src/medium-editor-md.js

```diff
@@ -26,10 +26,12 @@ module.exports = function (options, callback) {
     options.events = options.events || ["input", "change"];
     callback = callback || options.callback || function () {};
 
-    // Wait until the Medium Editor is inited
-    setTimeout(function() {
+    // Called by medium-editor during init
+    this.init = function(mediumEditorInstance) {
```

Comment:
So there can be more than one instance of medium-editor on the page, but ideally they'll be attached to different contenteditable elements.  If you attach multiple instances of medium-editor to the same element, I don't expect the behavior will be awesome (2 toolbars?) but I don't think that's a scenario we want.

However, medium-editor does allow you to pass multiple elements to it, and it will manage the state of each of those elements.  I'm not sure how well this is tested, or how widely it's being used, but it's technically supported.  If we wanted it to work correctly, we would need to somehow pass the element that the markdown is for to the callback (in addition to the markdown itself).

Something to think about, though I think just going with `elements[0]` will suffice for this pull request, what do you think?